app/templates/app/home.html

```diff
@@ -41,7 +44,7 @@ <h2 class="fs-3 card-title">{% trans "Explore" %}</h2>
             </li>
             <li class="m-2">
                 <a href="{% url 'languages' %}" class="btn btn-secondary">
-                    {% trans "Languages" %}
+                    {% trans "By language" %}
```

Comment:
If we go with this, for consistency we should also have "By subject field", and arguably "Per institution", etc.  So not sure if this is worth it. The "by" is in the preceding paragraph.